flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  description = "Decky development environment";
+  # pulls in the python deps from poetry
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    poetry2nix = {
+      url = "github:nix-community/poetry2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils, poetry2nix }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
```

Comment:
seemingly unnecessary usage of legacyPackages? should be able to call nixpkgs with the system argument instead.

Reply:
good point, fixed.